.github/workflows/test.yml

```diff
@@ -33,6 +34,23 @@ jobs:
     - name: Build mavlink-dump
       run: cargo build --verbose --bin mavlink-dump --features ardupilotmega
 
+  msrv:
+    needs: quick-tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+    - uses: actions/checkout@master
+    - uses: dtolnay/rust-toolchain@master
+      with:
+        toolchain: "1.60.0"
```

Comment:
Doesn't it make more sense to move the `build` step to use 1.60 ?

Reply:
Not necessarily. The usual approach is to run your tests and everything on stable (or nightly), and just run a check on your MSRV to check you haven't added dependencies or syntax that force downstream users to bump their compiler version.
But your primary target is still the newer compiler, and you want to opt in to any compiler warnings or errors that might not be present in the older compiler.

This is how I usually set this up- https://github.com/danieleades/monzo-lib/blob/main/.github/workflows/CI.yml (this is a pretty standard approach)

Reply:
Testing the beta compiler is definitely overkill...

Maybe a better approach is to add an MSRV target to the main build matrix?

Reply:
Thanks for the explanation!